src/02-crm/contacts.apib

```diff
@@ -66,6 +66,7 @@ Get a list of contacts.
                 + tags: prospect, expo (array[string])
                 + added_at: `2016-02-04T16:44:33+00:00` (string)
                 + updated_at: `2016-02-05T16:44:33+00:00` (string)
+                + canonical_url: `https://app.teamleader.eu/contact_detail.php?id=2a39e420-3ba3-4384-8024-fa702ef99c9f`
```

Comment:
PP : not sure if "canonical" makes sense in this context

Isn't his just the "application_url", "visit_url", "web_url", "page_url" or something?

Reply:
Canonical is wat Dietrich suggested based on what Drupal names this, but canonical probably references to the SEO meta tag "canonical_url" in Drupal.

Any preference?

Reply:
B: I thought we agreed on adding the url in a meta-data-like property either in 

1. `data: { }, metaData : { url: ... }`
2. `data: { _url: ... }`
3. `data: { @url: ... }`

And do we wrap inside a `links` attribute?

```
links {
    "self": "https://app.teamleader.eu/foo/bar/baz"
}
```

Reply:
@jenssegers thoughts?

Reply:
@stivni As mentioned here: https://github.com/teamleadercrm/api/pull/195#discussion_r204417716 I do like option 1 in combination with the `links` with maybe a more explicit url name like you mentioned in your first comment 👍

Reply:
Not sure if it belongs in the (REST) `links` section, that's more used for navigating through relations or pagination.

I'm also not sure if `canonical_url` is the correct name, but I kinda like it. We just need to make sure this is actually the correct term our url.

I'm afraid that once we start adding meta data to objects by prepending it with a `_` or `@`, we will end up discussing what belongs in meta data and what not.

Reply:
We went through a discussion and checked how major apis handled this. We came to the conclusion that `web_url` in the `data` object would be the most logical solution. (a bit more information in the commit message).